### DISCUSSION TITLE
Drag and Drop

### DISCUSSION THREAD

**Discussion by MinaMatta98** [sxyazi/yazi#424] [1. Q&A]
### Will you be willing to contribute this feature?

- [ ] Yes, I'll give it a shot

### Describe the solution you'd like

Hello,

Would you be capable of supporting drag and drop functionality?

It could be incorporated through: https://github.com/mwh/dragon

This would help complete the tui.

**Comment by MinaMatta98** [comment]
Note,

It currently looks like this when used with LF:

![image](https://github.com/sxyazi/yazi/assets/81748812/0ff2ae44-5e4a-42ae-9f85-0f0e30909735)

**Comment by sxyazi** [comment]
See https://github.com/sxyazi/yazi/discussions/327
